### ISSUE TITLE
One suggestion

### ISSUE THREAD

**Issue by jowbjj** [gigocabrera/MoneyLeash2#11]
A general informative of all accounts, with current balance, negative, etc., same as the version of Ionic 1.

**Comment by gigocabrera** [comment]
If you have a mockup or a more detailed view of what you're suggesting that would help. But I agree, I need to add a more informative screen view with all the accounts and info.

**Comment by jowbjj** [comment]
Of course, I'm going to look for some app that I have, in case I do not think I'll do a layout in Photoshop.
Thank you.
